index.js

```diff
@@ -18,8 +18,15 @@ var Promise     = require( 'es6-promise' ).Promise;
  *
  * @tested
  */
-function readDirectory( projectPath, credits ) {
+function readDirectory( projectPath, credits, seen ) {
   credits = credits || [];
+  seen = seen || [];
```

Comment:
I'm not sure what you mean.
On Nov 2, 2015 5:33 AM, "Stefan Judis" <notifications@github.com> wrote:

> In index.js
> <https://github.com/stefanjudis/credits/pull/4#discussion_r43614735>:
>
> >    credits = credits || [];
> > +  seen = seen || [];
>
> Can we align the =?
>
> I'm missing the eslint settings for this obviously.
>
> After that we're ready to go. :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/stefanjudis/credits/pull/4/files#r43614735>.
>